# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cups/no-man.patch.cross
# Copyright (C) 2009 - 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Converting man pages to HTML...
    cancel.1.gz...
/bin/sh: line 2: ./mantohtml: cannot execute binary file

  - Rene Rebe <rene@exactcode.de>

diff -Naur cups-1.4.5-VANILLA//Makefile cups-1.4.5/Makefile
--- cups-1.4.5-VANILLA//Makefile	2010-04-23 18:56:34.000000000 +0000
+++ cups-1.4.5/Makefile	2011-11-04 21:16:24.000000000 +0000
@@ -20,7 +20,7 @@
 # Directories to make...
 #
 
-DIRS	=	cups filter backend berkeley cgi-bin driver locale man monitor \
+DIRS	=	cups filter backend berkeley cgi-bin driver locale monitor \
 		notifier ppdc scheduler systemv test \
 		$(PHPDIR) \
 		conf data doc examples $(FONTS) templates
diff -Naur cups-1.4.5-VANILLA//ppdc/Makefile cups-1.4.5//ppdc/Makefile
--- cups-1.4.5-VANILLA/ppdc/Makefile	2010-04-24 02:56:34.000000000 +0800
+++ cups-1.4.5/ppdc/Makefile	2011-11-05 12:44:58.000000000 +0800
@@ -243,8 +243,9 @@
 	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o genstrings genstrings.o \
 		libcupsppdc.a ../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) \
 		$(DNSSDLIBS) $(COMMONLIBS) $(LIBZ)
-	echo Generating localization strings...
-	./genstrings >sample.c
+
+# 	echo Generating localization strings...
+# 	./genstrings >sample.c
